### ISSUE TITLE
XML to Proto online

### ISSUE THREAD

**Issue by YaroslavPodorvanov** [json-to-proto/json-to-proto.github.io#5]
On insert content we need detect is XML or JSON then convert to Protobuf

https://github.com/xml-to-proto/xml-to-proto.github.io

**Comment by eye-wave** [comment]
is this still open? i can work on it

**Comment by YaroslavPodorvanov** [comment]
> is this still open? i can work on it

Yes, you can in https://github.com/xml-to-proto/xml-to-proto.github.io

**Comment by Klebson-I** [comment]
Hi, can i be assign to it?

**Comment by YaroslavPodorvanov** [comment]
> Hi, can i be assign to it?

@Klebson-I, I created this page https://xml-to-proto.github.io/ to understand how many developers need this tool. You can build it yourself without waiting for me, as I’m currently working on other projects:
- https://github.com/u8views/go-u8views
- https://github.com/readytotouch/readytotouch
